src/components/AddPlaidBankAccount.js

```diff
@@ -128,9 +128,11 @@ class AddPlaidBankAccount extends React.Component {
             value: account.plaidAccountID, label: `${account.addressName} ${account.mask}`,
         }));
         const institutionName = lodashGet(this.props, 'plaidData.institution.name', '');
-        const selectedPlaidBankAccount = lodashGet(this.props, 'plaidData.selectedPlaidBankAccount', {});
         const {icon, iconSize} = getBankIcon();
 
+        // Show selected plaid bank account. If not set, check the props to see if we have one pre-selected
```

Comment:
when does that happen? Also, did you mean state instead of props?

Reply:
Well the state of `AddPersonalBankAccountPage`, but it's being passed into here as a prop.

Reply:
Sorry I'm still trying to wrap my mind around the merge with your PR. 

So when you select an account here, we set the state in AddPersonalBankAccount. I wanted to keep the bank account we selected when the user navigate back to the form.

Reply:
Then let's just pass selectedPlaidAccountID via props (add add it to the propTypes)

Reply:
Oh I just realized there is a bug (minor edge case) where this all gets messed up if someone is setting up a VBA (using BankAccounts.updatePlaidData) and also setting up a PBA. 

1. Try to create a PBA. 
2. Login into plaid, then selecting the test account that ends with `000`. Add an incorrect password and submit (creates a pending PBA)
3. Now go to your workspace settings. Choose "Connect bank account" to setup VBA. Log into plaid
4. Selected the bank account `1111`.
5. Navigate bank to the PBA, it will be set to `1111`. 
6. Click `0000`, go to next tab, the state forced it back to `1111`. We just need to run update here too

Reply:
We can just continue to use updatePlaidData here too?